logger/log.go

```diff
@@ -190,6 +200,46 @@ func (l *fileLogger) logDirect(label, format string, v ...interface{}) int {
 	return len(entry)
 }
 
+func (l *fileLogger) archivePurge(fname string) {
+	// Evaluate number of saved backups for purge
+	// l readlock held
+	var backups []string
+	lDir := filepath.Dir(fname)
+	lBase := filepath.Base(fname)
+	entries, err := os.ReadDir(lDir)
+	if err != nil {
+		l.logDirect(l.l.errorLabel, "Unable to read directory %q for log purge (%v), will attempt next rotation", lDir, err)
+		return
+	}
+	for _, entry := range entries {
```

Comment:
Easier way to collect these?

Reply:
Don't think so.  Since we allow the operator to put the log file in an arbitrary location, need to test each entry in the directory in several ways to make sure it is a legitimate log backup that we created.